app/static/js/minorAdminPage.js

```diff
@@ -1,43 +1,48 @@
 import searchUser from './searchUser.js'
-function emailAllInterested(){
-  // Read all student emails from the input as a string and put them in mailto format
-  let interestedStudentEmails =  $("#interestedStudentEmails").val();
-  // If there are any students interested, open the mailto link
-  if (interestedStudentEmails.length) {
-    const windowRef = window.open(`mailto:${interestedStudentEmails}`, '_blank');
+
+function emailMinorCandidates(studentEmails){
+  // If there are any students interested or declared, open the mailto link
+  if (studentEmails.length) {
+    const windowRef = window.open(`mailto:${studentEmails}`, '_blank');
     windowRef.focus();
     setTimeout(function(){
       if(!windowRef.document) {
           windowRef.close();
       }
     }, 500);
   } else {
-    msgFlash("No interested students to email.", "info")
+    msgFlash("No interested or declared students to email.", "info")
```

Comment:
Probably good to change this to "No students to email" since we don't know whether they emailing interested, declared, or both.